README.md

```diff
@@ -7,6 +7,7 @@ This project is a [balenaCloud](https://www.balena.io/cloud) stack with the foll
 - [Pi-hole](https://pi-hole.net/)
 - [PADD](https://github.com/pi-hole/PADD)
 - [Unbound](https://unbound.net)
+- [D-Bus]
```

Comment:
We probably don't need to advertise D-Bus here as it isn't a user-facing service.

Reply:
Yep, another one where I should have gone with my first idea - Fixing.  Thanks for the review!